dev/tools/grunt/configs/path.js

```diff
@@ -32,5 +32,6 @@ module.exports = {
         whitelist: 'dev/tests/static/testsuite/Magento/Test/Js/_files/whitelist/',
         blacklist: 'dev/tests/static/testsuite/Magento/Test/Js/_files/blacklist/',
         tmp: 'validation-files.txt'
-    }
+    },
+    generation: 'generated/'
```

Comment:
Any reason why it is called `generation` and not `generated`?

Reply:
Hi @krzksz 
I think "generation" is better, because it used a bit earlier, but inside the var directory.
I had been thinking as it should named, "generation" or "generated-files" and I decided as is as.

Reply:
Naming is ok.

Reply:
@andrewbess, alright thanks for the explanation.